doc/0000-ccl_note/main.tex

```diff
@@ -667,6 +674,84 @@ \subsubsection{Normalization of the power spectrum}
+  \frac{b_s}{2}*(s^2 - \langle s^2 \rangle) ~,
+\end{equation}
+where $s$ is the tidal field, and the mean values of both quadratic quantities are subtracted.
+The standard linear bias is $b_1$, and $(b_2,b_s)$ are the quadratic bias coefficients.
+The relevant galaxy-galaxy (for two galaxy samples, denoted by the second superscript) and galaxy-matter power spectra are then:
+\begin{align}
+  &P_{gg} = b_{1,1} b_{1,2} P_{d1d1} +
+               \frac{1}{2} (b_{1,1} b_{2,2} + b_{1,2} b_{2,1}) P_{d1d2} \notag\\
+               &~~~~ + \frac{1}{4} b_{2,1} b_{2,2} P_{d2d2} +
+               \frac{1}{2} (b_{1,1} b_{s,2} + b_{1,2} b_{s,1}) P_{d1s2} \notag\\
+               &~~~~ + \frac{1}{4} (b_{2,1} b_{s,2} + b_{2,2} b_{s,1}) P_{d2s2} +
+               \frac{1}{4} b_{s,1} b_{s,2} P_{s2s2}~,\\
+  &P_{gm} = b_1 P_{d1d1} + \frac{1}{2}b_2 P_{d1d2} + \frac{1}{2} b_s P_{d1s2}~,
+\end{align}
+where $P_{d1d1}$ corresponds to the chosen nonlinear matter power spectrum,
+and the other terms are one-loop PT expressions.
+
+\subsubsection{Intrinsic alignments}
+
+Following the ``TATT'' model (tidal alignment and tidal torquing) described in \cite{blazek19}, the galaxy intrinsic alignment field can be expanded:
+\begin{equation}
+    \gamma^{I}_{ij} = C_1 s_{ij} + C_2\left( s_{ik} s_{kj} - \frac{1}{3}\delta_{ij} s^2 \right) + C_{1\delta} (\delta s_{ij})~.
+\end{equation}
+Higher-order contributions will be included in future releases. The matter-intrinsic and intrinsic-intrinsic ($E$- and $B$-mode) power spectra are given by:
+\begin{align}
+\label{eq:dEtot}
+P_{\delta E} =& C_1 P_{d1d1} + C_{1\delta} \left[A_{0|0E} + C_{0|0E}\right]
++ C_2 \left[ A_{0|E2} + B_{0|E2}\right]~, \\
+\label{eq:EEtot}
+P_{EE} =&
+C_{1,1}C_{1,2} P_{d1d1} + (C_{1,1} C_{1\delta,2}+C_{1,2} C_{1\delta,1}) \left[A_{0|0E}+ C_{0|0E}\right] + C_{1\delta,1} C_{1\delta,2} A_{0E|0E} \notag\\
+&+ C_{2,1} C_{2,2} A_{E2|E2} + (C_{1,1} C_{2,2} + C_{1,2} C_{2,1}) \left[ A_{0|E2} + B_{0|E2}\right] \notag \\
+&+ (C_{1\delta,1} C_{2,2} + C_{1\delta,2} C_{2,1})D_{0E|E2}~,\\
+\label{eq:BBtot}
+P_{BB} =& C_{1\delta,1} C_{1\delta,2} A_{0B|0B} + C_{2,1} C_{2,2} A_{B2|B2} + (C_{1\delta,1} C_{2,2} + C_{1\delta,2} C_{2,1}) D_{0B|B2} ~,
+\end{align} 
+where, as above, $P_{d1d1}$ corresponds to the chosen nonlinear matter power spectrum,
+and the other terms are one-loop PT expressions.
+
+The normalization of the IA bias parameters can be flexibly specified. The convntion of \cite{blazek19} is
+\begin{align}
+    C_1(z) = - A_{1}(z) \frac{\bar{C}_1 \Omega_{\rm m} \rho_{\rm crit}}{D(z)}~, \\
+    C_{1 \delta}(z) = - A_{1 \delta}(z) \frac{\bar{C}_1 \Omega_{\rm m} \rho_{\rm crit}}{D(z)}~, \\
+    C_{2}(z) = A_{2}(z) \frac{5\bar{C}_1 \Omega^2_{\rm m} \rho_{\rm crit}}{\Omega_{\rm m,fid} D^2(z)}~.
+\end{align}
+The convention for $C_1(z)$ matches the normalization for $b_{\rm IA}$ used in the standard IA tracer (see Eq.~\ref{eq:bIA_norm}).
```

Comment:
How is this implemented in CCL? What is one passing?

Reply:
Clarified.